### PR TITLE
Update French.po 16.30 => 16.32

### PR DESCRIPTION
**Pull request by Need74**
Hi
I have just made this update (16.32) including the few latest new features since version 16.30 and also by tweaking the slightly French formulations (clumsy word-for-word translation) or sometimes errors in the meaning of the text in English compared to what is actually happens by activating the command (2 or 3 cases at most).

Aware that I have still made many small modifications which require verification, I remain attentive and in no way claim to have the absolute truth.

My pleasure is still great to work with WinMerge in parallel with Notepad++ and translating it is a real pleasure for me to share my passion for this program.

### CODE REVIEW

**Comment by sdottaka** [comment]
@Need74

Thank you for the PR!

@slolo2000

If you have no opinion about this PR, I will merge this PR.

File: Translations/WinMerge/French.po [hunk 1]
```diff
@@ -841,7 +841,7 @@ msgid "&View Whitespace"
 msgstr "Afficher les &espaces"
 
 msgid "View E&OL"
-msgstr "Afficher fin de ligne (E&OL)"
+msgstr "Afficher les carractères de &fin de ligne (EOL)"
```
Comment (slolo2000) [inline Translations/WinMerge/French.po]: Petite faute de frappe : carractères au lieu de caractères (avec un seul r)

**Review by slolo2000 — APPROVED** [review]
So many small changes but they are made with great intelligence and very good quality.
Many thanks for this code review.

File: Translations/WinMerge/French.po (hunk 1 above)
Comment (Need74) [reply Translations/WinMerge/French.po]: Merci pour la correction, et merci pour les mercis

Thanks for the correction, and thanks for the thanks

**Comment by sdottaka** [comment]
@Need74  @slolo2000 
Thank you!

**Pull request merged**